examples/all-clusters-app/esp32/sdkconfig.defaults

```diff
@@ -53,3 +53,9 @@ CONFIG_LWIP_HOOK_ND6_GET_GW_DEFAULT=y
 # Serial Flasher config
 CONFIG_ESPTOOLPY_FLASHSIZE_4MB=y
 CONFIG_ESPTOOLPY_FLASHSIZE="4MB"
+
+#disable Bluetooth modem sleep
```

Comment:
I think we should do this only for M5Stack and not for other boards

Reply:
done